app/src/main/java/fr/neamar/kiss/result/AppResult.java

```diff
@@ -46,15 +46,20 @@ public AppResult(AppPojo appPojo) {
 
     @Override
     public View display(final Context context, int position, View v) {
-        if (v == null)
-            v = inflateFromId(context, R.layout.item_app);
+        if (v == null) {
```

Comment:
i tested it on 4.4 and it seemed to work. I guess the problem appears on >4.4 ?
How about not checking if v==null and always set it to the appropriate layout?
Would that work?